File: Python/Program/Pembelian.py
class Pembelian:
    def __init__(self):
        self.nama_pembeli = ""            # Nama pembeli
        self.email_pembeli = ""           # Email pembeli
        self.metode_pembayaran = ""       # Metode pembayaran
        self.barang = []                  # List barang yang dibeli

    def set_pembelian(self, nama, email):
        self.nama_pembeli = nama          # Setter nama dan email pembeli
        self.email_pembeli = email

    def set_metode(self, metode):
        self.metode_pembayaran = metode   # Setter metode pembayaran

    def set_barang(self, item):
        self.barang.append(item)          # Tambah barang ke list

    def get_nama(self):
        return self.nama_pembeli          # Getter nama pembeli

    def get_email(self):
        return self.email_pembeli         # Getter email pembeli

    def get_metode(self):
        return self.metode_pembayaran     # Getter metode pembayaran

    def get_barang_list(self):
        return self.barang                # Getter list barang

File: PHP/Program/Pembelian.php
<?php
// Pembelian.php
class Pembelian {
    private $namaPembeli;
    private $emailPembeli;
    private $metodePembayaran;
    private $barang = []; // array of strings

    public function setPembelian(string $nama, string $email) {
        $this->namaPembeli = $nama;
        $this->emailPembeli = $email;
    }

    public function setMetode(string $metode) {
        $this->metodePembayaran = $metode;
    }

    public function setBarang(string $item) {
        $this->barang[] = $item;
    }

    public function setBarangArray(array $items) {
        $this->barang = $items;
    }

    public function getNama(): string {
        return $this->namaPembeli ?? '';
    }

    public function getEmail(): string {
        return $this->emailPembeli ?? '';
    }

    public function getMetode(): string {
        return $this->metodePembayaran ?? '';
    }

    public function getBarangList(): array {
        return $this->barang;
    }
}
